rectools/metrics/classification.py

```diff
@@ -430,15 +460,25 @@ def calc_classification_metrics(
         If unexpected metric is present in `metrics`.
     """
     k_map = defaultdict(list)
+    merged_debias = {}
     for name, metric in metrics.items():
         k_map[metric.k].append(name)
+        if not isinstance(metric, (ClassificationMetric, SimpleClassificationMetric)):
+            raise TypeError(f"Unexpected classification metric {metric}")
+        if metric.debias_config is not None and metric.debias_config.keyname not in merged_debias:
+            merged_debias[metric.debias_config.keyname] = make_downsample(merged, metric.debias_config)
```

Comment:
You can make the config frozen (which actually makes sense) and use it directly as a key